src/protocols/secure_channel/tests/TestCheckinMsg.cpp

```diff
@@ -98,7 +98,11 @@ void TestCheckin_Generate(nlTestSuite * inSuite, void * inContext)
         // caller to make sure the key is indeed valid.
         Aes128KeyHandle emptyKeyHandle;
         err = CheckinMessage::GenerateCheckinMessagePayload(emptyKeyHandle, counter, userData, outputBuffer);
+#if CHIP_CRYPTO_PSA
```

Comment:
Wait, why?  We're ensuring that generation _fails_ if CHIP_CRYPTO_PSA?

Why does it fail?  And do we not at bare mininum need a followup issue with a comment pointing to it?

But better yet would be to disable this entire test on the broken platform, not have weird ifdefs in the test itself.

Reply:
Test commented out and issue opened for future investigation. #28986